conanfile.py

```diff
@@ -89,7 +84,7 @@ def package_info(self):
         for component in ["viamsdk", "viamapi"]:
            self.cpp_info.components[component].set_property("cmake_target_name", "viam-cpp-sdk::{}".format(component))
            self.cpp_info.components[component].set_property("pkg_config_name", "viam-cpp-sdk-lib{}".format(component))
-           self.cpp_info.components[component].requires = ["grpc::grpc++", "protobuf::libprotobuf"]
+           self.cpp_info.components[component].requires = ["grpc::grpc++", "grpc::grpc++_reflection", "protobuf::libprotobuf"]
```

Comment:
It feels a little wrong to me that we need to reiterate library level dependencies known to cmake up here in the conan stuff. I'd expect that it would operate more at the package level and let CMake handle the library details. Can you elaborate on what's happening here and why?

Reply:
Thanks for the links and context, it makes more sense to me now.